app/api/v1/conversations.py

```
@@ -0,0 +1,38 @@
+    new_conv = Conversation(email=conversation.email)
+    session.add(new_conv)
+    await session.commit()
+    await session.refresh(new_conv)
+    return {
+        "id": new_conv.id,
+        "email": new_conv.email,
+        "started_at": new_conv.started_at
+    }
+
+@router.get("/conversations")
+async def get_conversations(session: AsyncSession = Depends(get_async_session)):
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)